src/axis.js

```diff
@@ -399,7 +271,171 @@ c3_chart_internal_fn.redrawAxis = function (transitions, isHidden) {
     transitions.axisSubX.call($$.subXAxis);
     // rotate tick text if needed
     if (!config.axis_rotated && config.axis_x_tick_rotate) {
-        $$.rotateTickText($$.axes.x, transitions.axisX, config.axis_x_tick_rotate);
-        $$.rotateTickText($$.axes.subx, transitions.axisSubX, config.axis_x_tick_rotate);
+        _rotateTickText($$.axes.x, transitions.axisX, config.axis_x_tick_rotate);
+        _rotateTickText($$.axes.subx, transitions.axisSubX, config.axis_x_tick_rotate);
     }
 };
+
+function _getTickValues(tickValues, axis) {
```

Comment:
Can you export these functions? As written in https://github.com/masayuki0812/c3/pull/929/files#r23514571, every function should be exported at this moment.